tests/coalaJSONTest.py

```diff
@@ -165,3 +165,29 @@ def test_show_language_bears_output_file(self):
         self.assertEqual(retval, 0)
         self.assertEqual(len(data['bears']), 2)
         self.assertFalse(stderr)
+
+    def test_output_file_overwriting(self):
+        with prepare_file(['#todo this is todo'], None) as (lines, filename):
+            args = (coala.main, 'coala', '--json', '-c', os.devnull, '-b',
+                    'LineCountTestBear', '-f', re.escape(filename),
+                    '--log-json')
+            execute_coala(*args)
```

Comment:
This invocation, and the one on line 180, are doing nothing.

Also, please move your new test method `test_output_file_overwriting` to appear under the similar test method `test_output_file`.